src/instructlab/eval/exceptions.py

```diff
@@ -58,3 +58,29 @@ class InvalidGitBranchError(EvalError):
     def __init__(self, branch) -> None:
         super().__init__()
         self.message = f"Invalid git branch: {branch}"
+
+
+class SDGPathNotFoundError(EvalError):
+    """
+    Error raised when the sdg path doesn't exist
+    Attributes
+        message         error message to be printed on raise
+        sdg_path        sdg path
```

Comment:
I think it would be good to make it an actual attribute, not just a constructor argument

Reply:
Fixed, thanks!